src/lib/apis/product-reviews.api.ts

```
@@ -0,0 +1,43 @@
+            headers: {
+                "Content-Type": "application/json",
+                Authorization: `Bearer ${token}`,
+            },
+        });
+
+        // Parse the response as JSON
+        const payload = (await res.json()) as ReviewsResponse;
+
+        // Check HTTP status
+        if (!res.ok) {
+            console.error("Failed to fetch reviews:", payload);
```

ESLint

[error] Unexpected console statement.

(no-console)